wsmaster/che-core-api-factory/src/main/java/org/eclipse/che/api/factory/server/FactoryLinksHelper.java

```diff
@@ -88,6 +93,24 @@ private FactoryLinksHelper() {}
               NAMED_FACTORY_ACCEPTANCE_REL_ATT);
       links.add(createWorkspaceFromNamedFactory);
     }
+
+    if (factory instanceof FactoryDevfileV2Dto) {
+      for (String additionalFile : additionalFilenamesProvider.get()) {
+        links.add(
+            createLink(
+                HttpMethod.GET,
+                uriBuilder
+                    .clone()
+                    .replacePath("api")
+                    .path(ScmFileResolverService.class)
+                    .path(ScmFileResolverService.class, "resolveFile")
+                    .queryParam("repository", repositoryUrl)
+                    .queryParam("file", additionalFile)
+                    .build(factoryId)
+                    .toString(),
+                "additional resource"));
```

Comment:
hello, does it mean all links will be named 'additional resource' ?

Reply:
I've tried and it's what it does

```
        {
            "href": "https://192.168.64.68.nip.io/api/scm/resolve?repository=https://github.com/che-samples/java-spring-petclinic/tree/devfilev2&file=.che/che-theia-plugins.yaml",
            "rel": "additional resource",
            "method": "GET"
        },
        {
            "href": "https://192.168.64.68.nip.io/api/scm/resolve?repository=https://github.com/che-samples/java-spring-petclinic/tree/devfilev2&file=.vscode/extensions.json",
            "rel": "additional resource",
            "method": "GET"
        }
```

could you use `additionalFile` in the name of the link ?

Reply:
ok